python/py-mupdf/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+name                py-mupdf
+
+# Keep synchronized with port:mupdf.
+github.setup        pymupdf PyMuPDF 1.21.1
+github.tarball_from archive
+
+revision            0
+categories          python graphics
+maintainers         nomaintainer
+license             AGPL-3+
+
+description         Python bindings for MuPDF, a lightweight PDF, XPS, and E-book viewer, renderer, and toolkit
+long_description    PyMuPDF adds Python bindings and abstractions to MuPDF, a lightweight PDF, XPS, and eBook viewer, renderer, and toolkit. Both PyMuPDF and MuPDF are maintained and developed by Artifex Software, Inc. \
+\
+MuPDF can access files in PDF, XPS, OpenXPS, CBZ, EPUB and FB2 (eBooks) formats, and it is known for its top performance and exceptional rendering quality. \
+\
+With PyMuPDF you can access files with extensions like .pdf, .xps, .oxps, .cbz, .fb2 or .epub. In addition, about 10 popular image formats can also be handled like documents: .png, .jpg, .bmp, .tiff, etc.
+
```

Comment:
please do "proper" line continuations for the long description so that lines in the Portfiles don't extend beyond 80 characters.